src/common-ui/components/Tags.js

```diff
@@ -18,13 +19,17 @@ const Tags = ({
     setInputRef,
     tagSearchValue,
     overview = false,
+    tag = false,
```

Comment:
This shouldn't really be derived in this view. Work it out in the container and just pass down a string maybe.